dependency_cache_test.go

```diff
@@ -560,6 +561,31 @@ func testDependencyCache(t *testing.T, context spec.G, it spec.S) {
 			Expect(io.ReadAll(a)).To(Equal([]byte("alternate-fixture")))
 		})
 
+		it("sets downloaded file name to uri's sha256", func() {
```

Comment:
test name does not seem to match behavior?

in the case that a dependency does not have a sha256 set, then the downloaded file name matches the artifact name

Reply:
Good catch. Fixed.